mountpoint-s3-fs/examples/fs_benchmark.rs

```diff
@@ -161,14 +161,16 @@ fn mount_file_system(
         mountpoint.to_str().unwrap()
     );
     let prefetcher_builder = Prefetcher::default_builder(client.clone());
-    let fs = S3Filesystem::new(
-        client,
-        prefetcher_builder,
-        runtime,
+    let superblock = Box::new(Superblock::new(
+        client.clone(),
         bucket_name,
         &Default::default(),
-        filesystem_config,
-    );
+        SuperblockConfig {
+            cache_config: filesystem_config.cache_config.clone(),
```

Comment:
Should we take these fields out of file system config? Happy to do it later though.